.travis.yml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+sudo: false
+
+os:
+- linux
+
+language: android
+
+android:
+  components:
+  - build-tools-22.0.1
+  - android-22
+
+env:
+  matrix:
+  - XWALK_CHANNEL=canary XWALK_VERSION=18.46.466.0
+
+before_install:
+- cd ..
+
+install:
+- git config --global user.email "minggangw@gmail.com"
```

Comment:
Will use intel email.  The other one is used as the account of Github, so I have to use the gmail account.